src/base/net/downloadmanager.cpp

```diff
@@ -210,6 +212,15 @@ bool Net::DownloadManager::deleteCookie(const QNetworkCookie &cookie)
     return static_cast<NetworkCookieJar *>(m_networkManager.cookieJar())->deleteCookie(cookie);
 }
 
+bool Net::DownloadManager::hasSupportedScheme(const QString &url)
+{
+    const QStringList schemes = QNetworkAccessManager().supportedSchemes();
```

Comment:
Instead of having this method static and create QNetworkAccessManager each time you call it I would drop static and use internal QNetworkAccessManager instance. We can still have static wrapper that just bypasses call to existing instance (singleton). There are no valid use cases outside of DownloadManager instance lifetime.

Reply:
OK, PR updated.